src/pages/index.tsx

```diff
@@ -760,7 +772,16 @@ class Map extends React.Component<PageProps, {}> {
       "restricted": features.features
         .filter(f => f.properties.activity === "restricted")
         .map(f => f.properties.length)
-        .reduce((acc, x) => acc + x, 0)
+        .reduce((acc, x) => acc + x, 0),
+      "off street": geojson.features.reduce((total, feature) => 
```

Comment:
May I suggest this in order to keep the same structure?

```suggestion
      "off street": geojson.features
        .filter(f => f.geometry.properties.type === "normal" && !f.geometry.properties.on_street)
        .map(f => f.geometry.properties.count)
        .reduce((acc, x) => acc + x, 0) * avgParkingLength,
      "electrique": geojson.features
        .filter(f => f.geometry.properties.type === "electrique")
        .map(f => f.geometry.properties.count)
        .reduce((acc, x) => acc + x, 0) * avgParkingLength,
      "communauto": geojson.features
        .filter(f => f.geometry.properties.type === "communauto")
        .map(f => f.geometry.properties.count)
        .reduce((acc, x) => acc + x, 0) * avgParkingLength
```